src/tools/PrizeTierController/utils/formatFormValuesFromPrizeTier.ts

```diff
@@ -1,39 +1,53 @@
-import { PrizeTierConfig, calculate } from '@pooltogether/v4-utils-js'
-import { EditPrizeTierFormValues } from '@prizeTierController/interfaces'
+import { calculate } from '@pooltogether/v4-utils-js'
+import { fallbackFormValues, fallbackFormValuesV2 } from '@prizeTierController/fallbacks'
+import { EditPrizeTierFormValues, PrizeTierConfigV2 } from '@prizeTierController/interfaces'
 import { BigNumberish } from 'ethers'
 import { formatUnits } from 'ethers/lib/utils'
 
 export const formatFormValuesFromPrizeTier = (
-  prizeTier: Partial<PrizeTierConfig>,
+  prizeTier: Partial<PrizeTierConfigV2>,
   decimals: BigNumberish,
-  options?: { round?: boolean }
+  options?: { round?: boolean; isV2?: boolean }
 ): Partial<EditPrizeTierFormValues> => {
-  const formValues: Partial<EditPrizeTierFormValues> = {
-    bitRangeSize: prizeTier.bitRangeSize.toString(),
-    expiryDuration: prizeTier.expiryDuration.toString(),
-    maxPicksPerUser: prizeTier.maxPicksPerUser.toString(),
-    endTimestampOffset: prizeTier.endTimestampOffset.toString(),
-    prize: undefined,
-    tiers: undefined
+  if (!!prizeTier) {
+    const formValues: Partial<EditPrizeTierFormValues> = {
+      bitRangeSize: prizeTier.bitRangeSize?.toString(),
+      expiryDuration: prizeTier.expiryDuration?.toString(),
+      maxPicksPerUser: prizeTier.maxPicksPerUser?.toString(),
+      endTimestampOffset: prizeTier.endTimestampOffset?.toString(),
+      prize: undefined,
+      tiers: undefined,
+      dpr: undefined
+    }
+    if (!!prizeTier.prize) {
+      const formattedPrize = formatUnits(prizeTier.prize, decimals)
+      formValues.prize = options?.round
+        ? parseFloat(parseFloat(formattedPrize).toFixed(Number(decimals) - 1)).toString()
```

Comment:
Is this rounding/conversion necessary? 
Reading this I'd expect `toFixed` to not actually chop any numbers off since you've already fit the number into the necessary amount of decimals with `formatUnits`

Reply:
Something in here might help as well <https://github.com/pooltogether/pooltogether-utilities/blob/master/src/utils/formatNumber.ts>

Reply:
This is unfortunately a side-effect of the precision issues from earlier (BigNumber division from `calculateTierPercentageForPrize()`). This rounding essentially ignores the last decimal altogether.

This can likely be cleaned up once `formatCombinedPrizeTier()` handles that deviant last decimal :)